packages/wix-ui-core/src/components/linear-progress-bar/LinearProgressBar.st.css

```
@@ -18,11 +18,17 @@
 .suffixIndication {}
 
 .barContainer {
+  -st-states: isButton;
   width: 100%;
   position: relative;
   height: 100%;
 }
 
+.barContainer:isButton {
+  border: none;
+  padding: 0;
+}
+
```

Actually I would just use a native `button` selector 😊 
Like this:
```suggestion
button.barContainer {
  border: none;
  padding: 0;
}

```

---

Makes sense. I'll change it